requirements.txt

```diff
@@ -23,3 +23,4 @@ rest-social-auth>=4.2.0
 six==1.12.0
 soupsieve==1.9
 SQLAlchemy==1.3.2
+PyGithub==1.54.1
```

Comment:
If you look carefully, you'll see that our dependencies are alphabetised :)

Reply:
Updated